apps/.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: ['plugin:react/recommended'],
+  extends: ['plugin:react/recommended', 'plugin:@next/next/recommended'],
```

Comment:
>plugin:@next/next/recommended [](http://example.com/codeflow?start=41&length=29)

Should this really be in the eslintrc file that applies to the whole apps directory? Or should it be in the eslintrc file just for the ssr-test-app directory, since it only applies to that one directory?

Reply:
You're right! Added an eslint file to the ssr app directory and moved the plugin.